### DISCUSSION TITLE
how to use custom color name?

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [tailwindlabs/tailwindcss#5133] [Help] [answered]
Hi. How to use custom color name in classname? 

```
theme: {
    extend: {
      colors: {
        parrot: "#bada55",
      },
    },
```

**Comment by huibizhang** [comment, marked as answer]
just using color normally~ example :
```html
<span class="text-parrot">
  I'm the text in color "parrot" !
</span>
```

you can see this, notice the HTML tag and Conifg tag.
https://play.tailwindcss.com/CTR3c6m7ke

**Reply by popuguytheparrot** [reply]
thanks, I would like to see illustrative examples in the documentation

**Reply by simonswiss** [reply]
Extending the `colors` object with the `parrot` color like you did will create **a lot** of new utilities for you:

- `bg-parrot` for [`background-color`](https://tailwindcss.com/docs/background-color)
- `text-parrot` for [`color`](https://tailwindcss.com/docs/text-color)
-  `border-parrot` for [`border-color`](https://tailwindcss.com/docs/border-color)
- `from-parrot` for the [`from` gradient color stop](https://tailwindcss.com/docs/gradient-color-stops#starting-color)

Same goes on with 

- `via-parrot` for the `via` gradient color stop
- `to-parrot` for the `to` gradient color stop
- `ring-parrot` for an outline box shadow color
-  ... and bunch more

We are not showing examples of using a custom color, but just know that whatever new color to generate, you'll be able to use these classes you would do with the normal default colors 👍

Looking at the examples for each of these (linked in this response) should let you imagine how to use your custom `parrot` color in all these scenarios!
